apps/sms/services/test/unit/mock_moz_mobile_connections.js

```diff
@@ -0,0 +1,24 @@
+/*exported MockMozMobileConnections */
+
+// The real navigator.mozMobileConnections is not a real array
+'use strict';
+
+var MockMozMobileConnections = {
+  0: {
+    iccId: 'SIM 1',
+    addEventListener: sinon.stub(),
```

Comment:
Oh, I was too lazy to stub the mock again in the test and we only use the mock for the test :p But you're right we never use sinon in mock.